src/pages/RegisterPage/RegisterPage.tsx

```diff
@@ -74,6 +75,7 @@ const RegisterPage: React.FC = (): React.ReactElement => {
               formState
             }) => (
               <TextField
+                helperText="Invalid email"
                 label="Password"
```

Comment:
This is what the UI currently looks like. Reminder to look at the UI every time you make a change and think through the changes. For forms, you should also be manually testing every change to make sure the behavior is what you expected.

This text that is related to errors is always up & is not shown in red. "Invalid email" should not show up before the user has typed anything into the input. Also, this text always says "invalid email" even when the email is valid. 

None of these messages should show unless the user makes a mistake and an error is triggered. When an error is triggered, the whole input, including the helper text, will be red courtesy of material ui. 

In order to do this ticket, you have to write the logic that ensures the helper text only appears when an error is triggered.

<img width="399" alt="Screen Shot 2022-10-20 at 1 00 40 PM" src="https://user-images.githubusercontent.com/48700332/197024064-d3627a8c-b766-4657-9057-db446907e78c.png">